server/db/models/index.js

```diff
@@ -9,25 +13,46 @@ const Product = require('./product')
  *    BlogPost.belongsTo(User)
  */
 
+ const ProductCategories = db.define('product_categories', {})
```

Comment:
Move OrderProducts into its own file. If you're not doing anything with ProductCategories, get rid of it, the association does this for you. If you are, move it to its own file.